model/Core/Classes/NamespaceMap.md

```diff
@@ -14,40 +14,25 @@ of ElementIDs to be used in SPDX content serialization in order to provide a mor
 human-readable and smaller serialized representation of the Elements.
 
 The content of a namespace map are suggestions for consistency in serialization but are not normatively enforced.
-Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content.
+When a serialization only contains the elements in "X", the collection content SHOULD (rather than MUST) utilize
+the namespace map.
 A serialization MAY choose to use the namespace map content.
 A serialization MAY choose to use prefixes and namespaces other than the namespace map content.
 A serialization MAY choose to use no prefixes at all and rather use the more verbose full ElementID IRIs.
 
-If utilized the namespace map set of prefixes and namespaces MUST be implemented in a given serialization 
-form (e.g., json-ld or xml) as specified in the binding rules specification for that serialization and 
-utilizing the appropriate inherent or custom specified mechanism for that serialization.
-The namespace map itself is also conveyed as native SPDX content to support clarity, transparency and 
-consistency independent of any particular serialization form.
-
-A given serialization payload (whether file or streaming) MUST NOT contain multiple namespace maps with conflicting mappings.
-Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content unless the single 
-serialization payload is intended to contain multiple peer collections each with their own namespace map.
-In this case the serializer MUST do one of the following:
-1) Diverge from the exact provided namespace mappings and utilize deconflicted mappings where appropriate.
-   This could be fully new prefix mappings as if the suggested namespace maps had not been provided.
-3) Use full ElementID IRIs instead of prefixed ElementID IRIs.
-
+If a serialization format supports prefixes or namespaces (e.g., JSON-LD context or XML XSD namespace), the namespace map
+must be represented in that format "native" to the serialization.
+The NamespaceMap itself will never be serialized as part of SPDX data if the serialization format support namespaces or prefixes.
+If the serialization format does not support prefixes, then the full URI's for the elements must be used and the namespace map will not be preserved.
+Any custom serialization format SHOULD implement namespaces in order to preserve the namespace map.
```

Comment:
I think this part should be described in [X-Collection.md](https://github.com/spdx/spdx-3-model/pull/491/files#diff-d87f35d07a17800a2a87a8d1f4a999b5816c4ab00435322d5f48db992bb7521a). The X-Collection should contain a description how it is expected to be created when deserializing a blob.